preset/preset_builtin.list

```diff
@@ -1,6 +1,6 @@
 <resources>
     <section name="PresetTemplate">
-        <integer name="VersionMajor" value="27" />
+        <integer name="VersionMajor" value="29" />
```

Comment:
This occurred to me. It's just an integer, but I'll remedy this since other changes are necessary.